README.md

```diff
@@ -1 +1,245 @@
+
+The private `@interface` is placed in the `.m` implementation file between the `#import` lines and the `@implementation` section and must be closed with an `@end` keyword before the `@implementation` keyword. Private properties can be added here without making them visible outside of the class. Private *methods*, however, don't need to be declared in the private interface section—the method names in the `@implementation` section make them visible within the current class file.
+
+So, an simple representation of that radio's interface in Objective-C might look something like this:
+
+```objc
+//  FISRadio.h
+
+#import <Foundation/Foundation.h>
+#import <CoreGraphics/CGBase.h>   // imports CGFloat
+
+@interface FISRadio : NSObject
+
+@property (nonatomic) CGFloat amFrequency;
+@property (nonatomic) CGFloat fmFrequency;
+
+@property (nonatomic) CGFloat swBal;
+@property (nonatomic) CGFloat tuning;
+
+@property (nonatomic) CGFloat tone;
+@property (nonatomic) CGFloat volume;
+
+@property (nonatomic) BOOL stereo;
+
+@property (nonatomic) CGFloat fmPreset1;
+@property (nonatomic) CGFloat fmPreset2;
+@property (nonatomic) CGFloat amPreset1;
+@property (nonatomic) CGFloat amPreset2;
+@property (nonatomic) CGFloat amPreset3;
+
+@end
+```
+
+## The Anatomy Of A Property
+
+Using the `@property` syntax, while it can feel verbose, is actually  a shorthand for generating the three components of a property: the instance variable, the "setter" method, and the "getter" method.
+
+#### The Instance Variable
+
+The **instance variable** (also "ivar") is the actual value that the property manages. It is both standard convention and implicit syntax that the instance variable is name after the property, but prepends an underscore character (`_`) for disambiguation. So, `_length` is the *instance variable* managed by the `length` property, and `_count` is the *instance variable* managed by the `count` property.
+
+**Note:** *Do* ***not*** *access an instance variable directly unless writing an initializer (common) or overriding a setter or getter (advanced). It best practice to use dot notation which implicitly calls the setter and getter methods for that property.*
+
+#### The Setter Method
+
+The **setter** is a method that is responsible for *writing to* (or "setting") the instance variable. It is both convention and implicit syntax that the setter is named after the property and prefixed with the word "set". The setter for the `length` property would be called `setLength:`, and the setter for the `count` property would be called `setCount:`. However, dot notation implicitly calls the setter method when used on the left side of an assignment operator:
+
+```objc
+// good style
+
+self.length = 10;
+```
+This is equivalent to:
+
+```objc
+// bad style
```

Comment:
I'd say "bad style; unnecessarily verbose" or something. There's nothing *wrong* with it, but if you *can* use dot syntax, you may as well.